website/content/docs/v0.14/Guides/storage.md

```diff
@@ -78,112 +80,33 @@ To see a live demo of this section, see the video below:
-
-- Under `/machine/sysctls`, add `vm.nr_hugepages: "512"`
-- Under `/machine/kubelet/extraMounts`, add `/var/local` like so:
-
-```yaml
-...
-extraMounts:
-  - destination: /var/local
-    type: bind
-    source: /var/local
-    options:
-    - rbind
-    - rshared
-    - rw
-...
-```
+This information is gathered from the Mayastor [documentation](https://mayastor.gitbook.io/introduction/quickstart/preparing-the-cluster).
+We need to set the `vm.nr_hugepages` sysctl and add `openebs.io/engine=mayastor` labels to the nodes which are meant to be storage nodes
+This can be done with `talosctl patch machineconfig` or via config patches during `talosctl gen config`.
 
-- Either using `kubectl taint node` in a pre-existing cluster or by updating `/machine/kubelet/extraArgs` in machine config, add `openebs.io/engine=mayastor` as a node label.
-If being done via machine config, `extraArgs` may look like:
+Some examples are shown below, modify as needed.
 
-```yaml
-...
-extraArgs:
-  node-labels: openebs.io/engine=mayastor
-...
-```
-
-### Deploy Mayastor
-
-Using the [Mayastor docs](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor) as a reference, apply all YAML files necessary.
-At the time of writing this looked like:
+Using gen config
 
 ```bash
-kubectl create namespace mayastor
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/deploy/moac-rbac.yaml
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/deploy/nats-deployment.yaml
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/csi/moac/crds/mayastorpool.yaml
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/deploy/csi-daemonset.yaml
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/deploy/moac-deployment.yaml
-
-kubectl apply -f https://raw.githubusercontent.com/openebs/Mayastor/master/deploy/mayastor-daemonset.yaml
+talosctl gen config my-cluster https://mycluster.local:6443 --config-patch '[{"op": "add", "path": "/machine/sysctls", "value": {"vm.nr_hugepages": "1024"}}, {"op": "add", "path": "/machine/kubelet/extraArgs", "value": {"node-labels": "openebs.io/engine=mayastor"}}]'
 ```
 
-### Create Pools
-
-Each "storage" node should have a "MayastorPool" that defines the local disks to use for storage.
-These are later considered during scheduling and replication of data.
-Create the pool by issuing the following, updating as necessary:
+Patching an existing node
 
 ```bash
-cat <<EOF | kubectl create -f -
-apiVersion: "openebs.io/v1alpha1"
-kind: MayastorPool
-metadata:
-  name: pool-on-talos-xxx
-  namespace: mayastor
-spec:
-  node: talos-xxx
-  disks: ["/dev/sdx"]
-EOF
+talosctl patch machineconfig -n <node ip> --config-patch '[{"op": "add", "path": "/machine/sysctls", "value": {"vm.nr_hugepages": "1024"}}, {"op": "add", "path": "/machine/kubelet/extraArgs", "value": {"node-labels": "openebs.io/engine=mayastor"}}]'
```

Comment:
`nit`: we can do `--immediate` here to avoid reboot

Reply:
good catch, fixing